test/tokencapgc.js

```diff
@@ -77,12 +81,11 @@ contract('TokenCapGC', accounts =>  {
     var tokenCapGC = await TokenCapGC.new();
     await tokenCapGC.setParameters(token.address,100);
```

Comment:
I think we should also make the Global Constraints into just Constraints, and remove `setParams` and everything.